app/views/queries/index.html.erb

```diff
@@ -13,6 +13,14 @@
       <th>Best bets</th>
       <th>Worst bets</th>
     </tr>
+
+    <tr class="table-header">
```

Comment:
I'm more in favour of a flexible template system with a _preferred_ approach, as documented by the style guide.